src/main/scala/mesosphere/marathon/core/launchqueue/LaunchQueue.scala

```diff
@@ -69,6 +69,7 @@ trait LaunchQueue {
 
   /** Remove all instance launch requests for the given PathId from this queue. */
   def purge(specId: PathId): Unit
```

Comment:
This internal method is not part of the API. Removing it shouldn't be a problem.